### DISCUSSION TITLE
Hi guys, does someone know when will ETCD v3.6.0 be released?

### DISCUSSION THREAD

**Discussion by JohnJAS** [etcd-io/etcd#16314] [Q&A] [answered]
I've checked readme and document, but I didn't find somewhere telling the release schedule. It will be helpful if someone knows when v3.6.0 will be released. Thanks in previous!

**Comment by jmhbnz** [comment, marked as answer]
Hey @JohnJAS - Thanks for your question, to the best of my knowledge at this point we do not have a planned release date.

Refer:
- [Changelog](https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.6.md#v360-tbd) where 3.6 is marked as TBD
- [Roadmap](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/roadmap.md#v360) outlining summary of main items at high level. 
- [Milestone](https://github.com/etcd-io/etcd/milestone/38 ) outlining remaining work in more detail

At this point I would suggest keeping an eye on the milestone for how the project is tracking against work required for the release.
